### ISSUE TITLE
 bookmark modal animation is incredibly slow

### ISSUE THREAD

**Issue by alexwykoff** [brave/browser-laptop#6748]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
When bookmarking a page via the keyboard shortcut, the modal animation is incredibly slow.

**Expected behavior:**
The modal should open almost instantly.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Tested on OS X.

- Brave Version (revision SHA):
0.13.0 preview 10 (c60b783)

- Steps to reproduce:
    1. Navigate to a site.
    2. Bookmarks the site via the keyboard combination.

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by bbondy** [comment]
I see your steps to reproduce, and i raise you a "Steps to not reproduce":
1. Clean your profile
2. Start the browser
3. Browse to brave.com
4. Wait for it to load
5. Command+D

Actual results:
Fades in fairly fast.

Any idea what I'm doing wrong?

**Comment by luixxiul** [comment]
This should have been solved with #6382

**Comment by bbondy** [comment]
The code with .bookmarkHanger doesn't exist anymore there, maybe check the history on the file to see when it was changed out after that.

**Comment by bbondy** [comment]
I don't think this is blocking, and we could use some more info here so I'm going to move to 0.13.1.

**Comment by luixxiul** [comment]
Is this not?

<img width="343" alt="screenshot 2017-01-20 13 43 46" src="https://cloud.githubusercontent.com/assets/3362943/22136222/85d167a4-df16-11e6-8b12-815662b01cf3.png">

**Comment by bbondy** [comment]
You're right @luixxiul I must have been searching the wrong repo or something.

**Comment by bsclifton** [comment]
With no owner and no solid STR, I don't think this should be included in 0.13.2.  Perhaps it is related to the # bookmarks that a user has? (and in that case, our site list => map change should help)

I'm going to mark as contributor backlog and tag as info needed

**Comment by luixxiul** [comment]
For a contributor: 

````
-    > * {
+
+    // Animate all direct descendants except for the bookmark bar
+    // PR #6382
+    > *:not(.bookmarkHanger) {
````

**Comment by bkilrain** [comment]
I'm up for the challenge!

**Comment by luixxiul** [comment]
@bkilrain Sweet! Please don't forget to include `Fix #6748` and `Fix #6380` in the commit message. Thanks in advance!

**Comment by luixxiul** [comment]
Fixed with https://github.com/brave/browser-laptop/pull/8443 (https://github.com/brave/browser-laptop/pull/8136)
